golem-worker-executor-base/src/services/oplog.rs

```diff
@@ -170,6 +184,22 @@ impl OplogService for OplogServiceDefault {
 
         entries
     }
+
+    async fn wait_for_replicas(&self, replicas: u8, timeout: Duration) -> bool {
+        let replicas = replicas.min(self.replicas);
+        match self
+            .redis
```

Comment:
Wow, looks like stumbled on the correct API here if Redis does something similar. 😄